### ISSUE TITLE
bug: `addSQLiteSuffix` skips databases without `.db` extension

### ISSUE THREAD

**Issue by robingenz** [capacitor-community/sqlite#200]
Hi jepiqueau, 
first of all thank you for this great plugin.
I am currently trying to migrate from `cordova-sqlite-storage` to this plugin and ran into a problem.

**Describe the bug**

My databases do not have a `.db` extension.
Examples: `mfapp.logs`, `mfapp.data`

This causes them to be skipped in `addSQLiteSuffix`: https://github.com/capacitor-community/sqlite/blob/2a04454d38f3ff949ad89d5c076628ab02926745/android/src/main/java/com/getcapacitor/community/database/sqlite/SQLite/UtilsMigrate.java#L46

My code:

```js
private async migrateCordovaSqliteDb(): Promise<void> {
    const result = await this.sqlite.getMigratableDbList('default');
    const dbNames = (result.values || []).filter(
        (value: string) => !!value && value.includes('mfapp.'),
    );
    await this.sqlite.addSQLiteSuffix('default', dbNames);
    await this.sqlite.deleteOldDatabases('default', dbNames);
}
```

**To Reproduce**

1. Create a database with `cordova-sqlite-storage` without `.db` extension.
2. Try to migrate the database using this plugin and the `addSQLiteSuffix` method (see code example above).

**Expected behavior**

I expect that databases without the `.db` file extension can be migrated as well.

**Screenshots**

**Smartphone (please complete the following information):**

- Device: OnePlus 7T
- OS: Android 11
- Browser Chrome
- Version 96.0

**Additional context**

Plugin version: 3.3.1
Capacitor Android version: 3.3.2
Capacitor Core version: 3.3.2

**Comment by jepiqueau** [comment]
@robingenz the cordova database must have a db extension to be migrated with `addSQLiteSuffix` this was implemented like this to avoid the transfer of non-database.

**Comment by robingenz** [comment]
@jepiqueau Thank you for your answer!
I understand this and can see that this is a reasonable default behavior.
However, in my particular case, I clearly specify the database names I want to migrate.
At this point, I don't think there is any reason for this restriction anymore.
Or am I missing something?

What options are there for me now to migrate my databases?

**Comment by jepiqueau** [comment]
@robingenz i will have a look

**Comment by jepiqueau** [comment]
@robingenz when transferred the database will have a SQLite.db suffix

**Comment by robingenz** [comment]
> when transferred the database will have a SQLite.db suffix

That is not a problem.
My main problem is that I cannot currently migrate a database without a `.db` extension, even though I know the exact name of the database.

**Comment by jepiqueau** [comment]
@robingenz that is fixed in 3.3.2

**Comment by robingenz** [comment]
@jepiqueau Thank you very much! Awesome work! I just tried it and it works without any problems.
